clients/include/rocblas_arguments.hpp

```diff
@@ -78,9 +78,9 @@ struct Arguments
     int32_t  solution_index;
     uint32_t flags;
 
-    char function[64];
-    char name[64];
-    char category[64];
+    char         function[64];
+    char         name[64];
+    mutable char category[64];
```

Comment:
Could comment here why this is mutable.

Reply:
It is no longer need to be `mutable`. Instead, in the one place where `Arguments` is `const` but we need to modify `category`, we use `const_cast`.